README.md

```diff
@@ -9,60 +9,49 @@ cp ./config/config.yaml.example ./config/config.yaml
 cp ./config/iss.json.example ./config/iss.json
 cp ./.env.example ./.env
 ```
+
 no further editing to the above files is required for running the stack locally.
 
 ## Starting the full stack with LS-AAI-mock
 
-To bootstrap the *full stack* of `storage-and-interfaces` services use
-the file `docker-compose.yml`. Note that this requires a running [`LS-AAI-mock`](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock) service. To configure the LS-AAI-mock service follow the instructions below.
-
-Add the following line to `/etc/hosts` file:
-
-```
-127.0.0.1 aai-mock
-```
+To bootstrap the *full stack* of `storage-and-interfaces` services use the file `docker-compose.yml`. Note that this requires a running [`LS-AAI-mock`](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock) service. To configure the LS-AAI-mock service follow the instructions below.
 
 First clone the [startet-kit-lsaai-mock](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock) repo.
```

Comment:
Maybe here mention that a specific branch in the `ls-aai` repo needs to be checked out.

Reply:
No, a PR will be generated from that branch now when someone else than me have verified that it works.